File: KuantumKaosOdev-java/src/VeriPaketi.java
public class VeriPaketi extends KuantumNesnesi {

    public VeriPaketi(String id) {
        super(id, 1);
    }

    @Override
    public void AnalizEt() throws KuantumCokusuException {
        // Stabiliteyi getter/setter ile yönetiriz
        double yeniDeger = getStabilite() - 5;
        setStabilite(yeniDeger);
        System.out.println("Veri içeriği okundu.");
    }
}

File: KuantumKaosOdev-csharp/VeriPaketi.cs
using System;

namespace KuantumKaos
{
    public class VeriPaketi : KuantumNesnesi
    {
        public VeriPaketi(string id) : base(id, 1) // Tehlike seviyesi düşük
        {
        }

        public override void AnalizEt()
        {
            Stabilite -= 5;
            Console.WriteLine("Veri içeriği okundu.");
        }
    }
}